cmd/immuclient/immuc/print.go

```diff
@@ -43,7 +40,7 @@ func PrintKV(key []byte, md *schema.KVMetadata, value []byte, tx uint64, verifie
 		}
 
 		str.WriteString(fmt.Sprintf("value:		%s \n", value))
-		str.WriteString(fmt.Sprintf("hash:		%x \n", hash))
```

Comment:
I think from immuclient the hash may not be providing something useful. Displaying it will require immuclient to receive the header (or at least the version of the tx), which may imply changing the grpc endpoints in the server